controllers/storagecluster/reconcile.go

```diff
@@ -370,6 +370,9 @@ func (r *StorageClusterReconciler) reconcilePhases(
 				&ocsCephCluster{},
 				&ocsCephBlockPools{},
 				&ocsCephFilesystems{},
+				&ocsCephNFSBlockPool{},
+				&ocsCephNFS{},
+				&ocsCephNFSService{},
```

Comment:
Two things:
1. Following the conventions from above, this should be `ocsCephNFSes` and `ocsCephNFSServices` (plural).
2. Since OCS creates a service for each CephNFS, why not include the service creation and deletion as part of creating/deleting CephNFSes?

Reply:
> Two things:
> 
> 1. Following the conventions from above, this should be `ocsCephNFSes` and `ocsCephNFSServices` (plural).

It now follows this pattern.

> 2. Since OCS creates a service for each CephNFS, why not include the service creation and deletion as part of creating/deleting CephNFSes?

It will lead to a very complicated `ensureCreated` and `ensureDeleted` function. 
It is much cleaner with two different structs.